test/unittest.cpp

```diff
@@ -24,18 +26,32 @@ bool TestMemoryLeaks() {
 	return test_memory_leaks;
 }
 
+bool SummarizeFailures() {
+	return summarize_failures;
+}
+
 } // namespace duckdb
 
 int main(int argc, char *argv[]) {
 	duckdb::unique_ptr<FileSystem> fs = FileSystem::CreateLocal();
 	string test_directory = DUCKDB_ROOT_DIRECTORY;
+	const char *filename("failures_summary.txt");
```

Comment:
Where the failures should be saved to?
Should that be configurable?
Currently if you do
```
./build/release/test/unittester
```
I guess this will be in your main duckdb folder.

And this means running unittester multiple times will (if I understand this correcty) not delete previous results. I guess both sequential but also parallel are somewhat relevant.

I would think adding UUID can be handy and also allowing to change base folder maybe? or use the folder used by unittester already? or use a variation (|| "_error_summaries") on  the folder used to store data?